src/librustdoc/core.rs

```diff
@@ -458,6 +458,11 @@ crate fn create_resolver<'a>(
     resolver.clone()
 }
 
+/// This where the rustc specific passes should be placed.
+fn run_rustc_checks(tcx: TyCtxt<'_>) {
+    rustc_passes::stability::check_unused_or_stable_features(tcx);
+}
```

Comment:
I'm not sure why you're still treating this differently. If you're going to group the rustc-specific things together, I would put this near https://github.com/rust-lang/rust/blob/3127a8d9ec95ec11a4cf039305e7bbc54a436b6b/src/librustdoc/core.rs#L486-L499. I would prefer not to make it a separate function so it's easier to see what's going on.

Reply:
Moved there!